src/screens/SignupForm/styles.module.scss

```diff
@@ -3,13 +3,15 @@
 .signup-form {
   background-color: $white-main;
   border-top: 6px solid $blue;
-  display: flex;
-  flex-direction: column;
-  margin: 30px auto;
+  margin-top: 30px;
```

Comment:
avoid use margin-top, use margin-bottom in the brother element, or padding-top in pather element if necessary